<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sendByDialog</class>
 <widget class="QMainWindow" name="sendByDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="LEdialogPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="BselectDialogPath">
        <property name="text">
         <string>选择日志路径</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>发送基准</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="RBsendBaseOnReceive">
         <property name="text">
          <string>接收记录</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RBsendBaseOnSend">
         <property name="text">
          <string>发送记录</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>发送速度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="CBrepeadSpead">
        <item>
         <property name="text">
          <string>1倍</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2倍</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4倍</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8倍</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16倍</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32倍</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="Bconfirm">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Bcancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>272</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
